### ISSUE TITLE
Load More Button not showing

### ISSUE THREAD

**Issue by nidhie2logy** [kop/yii2-scroll-pager#46]
Hello,

I have used scroll pager and pageSize is 1 and i have 5 records but after click on show more button second record shown and after that button is disappear.
My code is below:

```
echo ListView::widget( [
    'dataProvider' => $dataProvider,
    'itemView' => function ($model, $key, $index, $widget) {
      
        return $this->render('_item',['model' => $model]);
        
        // or just do some echo
        // return $model->title . ' posted by ' . $model->author;
    },
    'id' => 'my-listview-id',
    'pager' => ['class' =>\kop\y2sp\ScrollPager::className(),            
                 'enabledExtensions' => [
                \kop\y2sp\ScrollPager::EXTENSION_TRIGGER,
                \kop\y2sp\ScrollPager::EXTENSION_SPINNER,
                \kop\y2sp\ScrollPager::EXTENSION_NONE_LEFT,
                \kop\y2sp\ScrollPager::EXTENSION_PAGING,
            ],],
] );
```

Please Help.

THanks

**Comment by pliszka** [comment]
Same problem I copy paste the readme.md example but it's not working. After the Load more "button" was clicked once it's not showing up any more once you reload page.

**Comment by pliszka** [comment]
found a work around: 

````
'eventOnRendered' => 'function() {this.next();}',
````

Total working example:
````
<?= ListView::widget([
    'id' => 'list-view-scroll-pager',
    'dataProvider' => $dataProvider,
    'itemView' => function ($model, $key, $index, $widget) {
        return $this->render('_list_item',['model' => $model]);
    },
    'itemOptions' => [
        'class' => 'item',
    ],
    'pager' => [
        'class' => \kop\y2sp\ScrollPager::className(),
        'eventOnRendered' => 'function() {this.next();}',
    ]
]); ?>
````

For me this issue is connected with the trigger of Load more items button, it's appeasr only if the page could scroll down to the bottom so if you hit load more and this download less elements that actually could expand the page to have the scroll bar this is not showing.

**Comment by nidhie2logy** [comment]
Thanks this fix work for me but summary is not updating.its still showing Showing 1-1 of 5 items

**Comment by kop** [comment]
@nidhie2logy, sorry for the late response.

I've tested code given for you and it works just fine (after I've added missing `'itemOptions' => ['class' => 'item']` option).

I'm unable to reproduce this bug, so I suppose it's something related to your exact markup.

As for auto-updating $summary, it's not implemented yet and is tracked in #19.

**Comment by rachidaitmallouk** [comment]
<?=
                    ListView::widget([
                      'dataProvider' => $proData,
                       'itemView' => '_post',
                       'summary'=>'',
                       'pager' => [
                      'class' => \kop\y2sp\ScrollPager::className(),
                      'eventOnRendered' => 'function() {window.setTimeout(function() {$(window).scroll();}, 500);}',                    
                        ],
                       'itemOptions' => ['class' => 'col-sm-3'],
                       
                                      
                        ]);?>
Not display results?
